File: P2N_Pet_FrontEnd/P2N-Pet/projects/admin/src/app/models/supplier.ts
export class Supplier{
    public Id: number;
    public Name: string;
    public Address: string;
    public Phone: string;
    public Email: string;
    public StatusText: string;
    public CreateUserName: string;
    public CreateDate: string;
    public UpdateUserName: string;
    public UpdateDate: string;
}

export class SupplierCondition {
    Name: string;
    Phone: string;
    Email: string;
    Status: string;
  
    constructor() {
      this.Name = '';
      this.Phone = '';
      this.Email = '';
      this.Status = '0';
    }
  }

File: P2N_Pet_BackEnd/P2N_Pet_API/P2N_Pet_API/Database/PetShopModels/Supplier.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace P2N_Pet_API.Database.PetShopModels
{
    public partial class Supplier
    {
        public Supplier()
        {
            Pets = new HashSet<Pet>();
        }

        public ulong Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public int? Status { get; set; }
        public ulong? Createuser { get; set; }
        public DateTime? Createdate { get; set; }
        public ulong? Updateuser { get; set; }
        public DateTime? Updatedate { get; set; }

        public virtual ICollection<Pet> Pets { get; set; }
    }
}
